src/internal/internalAPIs.ts

```diff
@@ -9,11 +9,18 @@ GlobalVars.handlers["fullScreenChange"] = handleFullScreenChange;
 GlobalVars.handlers["backButtonPress"] = handleBackButtonPress;
 GlobalVars.handlers["beforeUnload"] = handleBeforeUnload;
 GlobalVars.handlers["changeSettings"] = handleChangeSettings;
-GlobalVars.handlers["getConversationId"] = handleConversationId;
+GlobalVars.handlers["onStartConversation"] = handleStartConversation;
```

Comment:
NIT: For consistency consider following the existing naming pattern here for the event + handler. Either:
```
GlobalVars.handlers["startConversation"] = handleStartConversation;
GlobalVars.handlers["closeConversation"] = handleCloseConversation;
```

Or:
```
GlobalVars.handlers["onStartConversation"] = handleOnStartConversation;
GlobalVars.handlers["onCloseConversation"] = handleOnCloseConversation;
```

Reply:
for consistency use this 
GlobalVars.handlers["startConversation"] = handleStartConversation;
GlobalVars.handlers["closeConversation"] = handleCloseConversation;